cumulusci/core/config/project_config.py

```diff
@@ -380,13 +396,11 @@ def repo_commit(self):
                         ):
                             return parts[0]
 
-    def get_github_api(self, owner=None, repo=None):
-        return get_github_api_for_repo(
-            self.keychain, owner or self.repo_owner, repo or self.repo_name
-        )
+    def get_github_api(self, url=None):
```

Comment:
Changing this function signature needs to be called out as a breaking change in release notes.